test/blas1_test.hpp

```diff
@@ -0,0 +1,175 @@
+#ifndef BLAS1_TEST_HPP_DFQO1OHP
+#define BLAS1_TEST_HPP_DFQO1OHP
+
+#include <cmath>
+#include <complex>
+#include <cstdlib>
+#include <iomanip>
+#include <iostream>
+#include <vector>
+
+#include <gmock/gmock.h>
+#include <gtest/gtest.h>
+
+#include <interface/blas1_interface_sycl.hpp>
+
+using namespace cl::sycl;
+using namespace blas;
```

Comment:
Maybe it makes sense to remove `using namespace cl::sycl;`, but as a part of the project I think it makes sense to assume that we are using this library's namespace,